app/presenters/detailed_guide_presenter.rb

```diff
@@ -23,6 +23,12 @@ def related_navigation
     nav
   end
 
+  def logo
+    image = content_item.dig("details", "image")
+    return unless image
+    { path: image["url"], alt_text: "European structural investment funds" }
```

Comment:
Is there a way that this alt text can be put into the content item?

Reply:
@fofr this bothered me too, I don't understand why we return a url in the content item data and not some alt text. It's not currently presented to the Publishing API https://github.com/alphagov/whitehall/blob/master/app/presenters/publishing_api/detailed_guide_presenter.rb#L64

Reply:
Is it difficult to add? ("Image" is a little ambiguous too)

Reply:
There are 6 detailed guides in the content store using one of the two variations of this logo. There are no other detailed guides using other images so the problem is relatively limited.
I will open a PR in Whitehall to add the alt text, in the mean time, to satisfy the fairly urgent ZD issue, hardwired alt text will be consistent with the previous version of this format.

Reply:
Actually looking back at the previous version https://github.com/alphagov/government-frontend/blob/061c742a74af77c6fc55b80ac7ed42e0426dac13/app/views/content_items/detailed_guide.html.erb#L8 I don't see mention of alt text but the national archives version implies it was there: http://webarchive.nationalarchives.gov.uk/20171012232401/https://www.gov.uk/guidance/england-2014-to-2020-european-structural-and-investment-funds
very confusing.

Reply:
Ah - https://apidock.com/rails/ActionView/Helpers/AssetTagHelper/image_tag the name part of the source is used. I'll add a PR to do this until we get this data from Whitehall.